client/src/components/game/lab2/components/replay.js

```diff
@@ -0,0 +1,168 @@
+import React, { Component } from "react";
+import Button from "./header/buttons/button";
+import "./gameStyle.css";
+
+/*
+  Class for replay screen and to allow the user to replay the game
+*/
+class Replay extends Component {
+  //Constructor to hold state information
+  constructor(props) {
+    super(props);
+    this.state = {
+      scorePopup: false,
+      gameMode: null
+    };
+  }
+
+  //Renderer for application
+  render() {
+    // eslint-disable-next-line
+    this.state.gameMode = "default";
+
+    //Handles the first click
+    const clickFirst = () => {
+      this.props.changeGameColors(this.props.colors);
+      this.props.resetOption();
+      this.props.enterInfoState();
+    };
+
+    const clickSecond = () => {
+      this.props.changeGameColors(this.props.colors);
+      this.props.resetOption();
+      this.props.enterSecondInfoState();
+    };
+
+    const clickThird = () => {
+      this.props.changeGameColors(this.props.colors);
+      this.props.resetOption();
+      this.props.enterThirdInfoState();
+    };
+
+    return (
+      <div>
+        {/*
+         <div className='replay_screen'>
+           <div className='left'>
+             <ScoreBreakdown />
+             <PreviousGames score={this.props.score} mode={this.props.gameMode}/>
+             <ScoreComparison score={this.props.score} mode={this.props.gameMode}/>
+           </div>
+         </div>
+        */}
+        <div style={{ marginTop: "75px" }}>
+          <div>
+            {this.props.score > 0 ? (
+              <p className="timeEnd">Good job! Your score information is:</p>
+            ) : (
+              <p className="timeEnd">
+                Better luck next time! Your score information is:
+              </p>
+            )}
+            <div>
+              <table style={{ backgroud: "white" }} className="center">
```

Comment:
I'm not sure if it's just me but the score results table isn't centered
<img width="1179" alt="Screen Shot 2021-03-08 at 4 56 09 PM" src="https://user-images.githubusercontent.com/54957497/110387143-32db8e80-802f-11eb-8444-d93d83ef69c0.png">

Reply:
yep its off to the left